environment-cpp.yml

```diff
@@ -0,0 +1,13 @@
+name: xeus-kernel
+channels:
+  - https://repo.prefix.dev/emscripten-forge-dev
+  - conda-forge
+dependencies:
+  - xeus-cpp
+  - symengine
+  - xtensor-blas
+  - xsimd
+  - xwidgets
+  - xcanvas
+  - pip:
+      - ipycanvas==0.11
```

Comment:
Looks like `ipycanvas` is on conda-forge, so maybe no need to install it via `pip`? https://anaconda.org/conda-forge/ipycanvas

Also do we need a pin on `0.11`, or could a newer version be used?

Reply:
So xcanvas is updated only enough to work with ipycanvas 0.11 and hence the pin here

Reply:
And then we could look into making `xcanvas` support the latest `ipycanvas` separately.

Reply:
Perfect, let's do that. xcanvas needs some work on our end anyways.